javascript/javascript3/week1/WeatherApp/index.js

```diff
@@ -0,0 +1,118 @@
+const input = document.getElementById('city_name');
+const button = document.getElementById('btn');
+const msg = document.getElementById('errorMsg')
+
+button.addEventListener('click', () => {
+    if (input.value) {
+        const cityName = input.value.toLowerCase();
+        url = 'https://api.openweathermap.org/data/2.5/weather?q=' + cityName + '&appid=19a22c19916855a97938c4084c52be58';
+        fetch(url)
+            .then(response => response.json())
+            //.then(data=>console.log(data))
+            .then(data => renderCityWeatherInfo(data));
+    } else {
+        msg.innerHTML = 'Please enter the city';
+        return false;
+    }
+});
+
+function renderCityWeatherInfo(data) {
+    msg.innerHTML = " "
+    input.value = ""
+    const location = document.getElementById('location');
+    location.innerHTML = data.name;
+
+    const weatherIcon = document.getElementById('weather_icon');
+    weatherIcon.src = 'http://openweathermap.org/img/w/' + data.weather[0].icon + '.png';
+
+    const temp = document.getElementById('temperature');
+    temp.innerHTML = Math.floor(data.main.temp - 273) + '&#176' + 'C';
+
+    const description = document.getElementById('desc');
+    description.innerHTML = data.weather[0].description;
+
+    const wind = document.getElementById('wind_speed');
+    wind.innerHTML = 'Wind speed: ' + data.wind.speed + ' m/s';
+
+    const high = document.getElementById('max_temp');
+    high.innerHTML = 'High: ' + Math.floor(data.main.temp_max - 273) + '&#176' + 'C';
+
+    const Low = document.getElementById('min_temp');
```

Comment:
Remember to stick to your naming convention here. Usually capitalized variables, such as Low, is reserved for classes.
```suggestion
    const low = document.getElementById('min_temp');
```